UnitsNet.Tests/GeneratedCode/TestsBase/AccelerationTestsBase.g.cs

```diff
@@ -1211,6 +1211,10 @@ public void ToString_ReturnsValueAndUnitAbbreviationInCurrentCulture()
                 Assert.Equal("1 nm/s²", new Acceleration(1, AccelerationUnit.NanometerPerSecondSquared).ToString());
                 Assert.Equal("1 g", new Acceleration(1, AccelerationUnit.StandardGravity).ToString());
             }
+            catch
+            {
+                throw;
```

Comment:
Although there should never be an exception in a ToString class, I want it to show up in the tests if there is!